jenkins/shared-library/vars/createMonorepoMultibranchJobs.groovy

```diff
@@ -49,8 +50,12 @@ void generatePackageMultibranchJobs(Object buildConfiguration, Map jobConfigurat
 		jobConfiguration.displayName = build.name
 		jobConfiguration.packageFolder = Paths.get(jobConfiguration.repositoryName.toString()).resolve(build.path).toString()
 		jobConfiguration.daysToKeep = build.logRotation
+		renameJobIfDisplayNamePresent(jobConfiguration.fullBranchFolder, jobConfiguration.displayName, jobConfiguration.jobName)
```

Comment:
do we need this ?

Reply:
Thought it would be good to keep job history.
If you dont think its necessary will remove it.  It will make it cleaner also.

Reply:
that was my guess, just wanted to confirm